index.bs

```diff
@@ -369,6 +376,7 @@ All {{FontIterator}} objects contain an internal <dfn attribute for=FontIterator
 The <dfn iterator for=FontIterator>asynchronous iterator initialization steps</dfn> for {{FontIterator}} are as follows:
 
 1. Set [=/this=]'s {{FontIterator/[[FontList]]}} to a new empty [=/queue=].
+1. Obtain a <a href="https://infra.spec.whatwg.org/#list-sort-in-ascending-order">list sorted in ascending order</a> of each local font |font|.
```

Comment:
I think you can write `[=list/sort in ascending order|list sorted in ascending order=]`

But... you don't explain how to sort, i.e. that this is by postscriptName 

Also, this doesn't align with the next step, which is enumerating the system's fonts. |font| isn't used in the algorithm, etc. and there's no clear type for it. Can you rewrite this to be more like code? Maybe at a high level:

1. Enumerate system fonts
    1. Convert each to a font representation
    1. Append to list
1. Let list be list, sorted by postscript name
1. Enqueue each item of the list to the iterator's fontlist

Maybe the last steps can be combined?